### PR TITLE
LPS-181946 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-181946](https://issues.liferay.com/browse/LPS-181946)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 1]
```diff
@@ -4611,7 +4611,7 @@ definition {
 
 		Button.clickPublish();
 
-		Alert.viewErrorMessage(errorMessage = "The Start node must have an outgoing transition");
+		Alert.viewErrorMessage(errorMessage = "The Task node must have an incoming transition");
```
Comment (epark18) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:4614]: By changing the error message; the test isn't really confirming _StartNodeMustHaveOutgoingTransition_ anymore. Is there a way to change the test steps to assert the original error?

**Review by epark18 — COMMENTED** [review]
@davisaints The test may be passing now, but I think it moved too far away from the original point/assertion the test needs to make.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase (hunk 1 above)
Comment (davisaints) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:4614]: You are sure. I hadn't noticed that.

**Pull request closed without merging**